src/plugins/chart_expressions/expression_xy/common/types/expression_functions.ts

```diff
@@ -155,16 +180,62 @@ export interface LabelsOrientationConfig {
 
 // Arguments to XY chart expression, with computed properties
 export interface XYArgs {
-  title?: string;
-  description?: string;
   xTitle: string;
   yTitle: string;
   yRightTitle: string;
   yLeftExtent: AxisExtentConfigResult;
   yRightExtent: AxisExtentConfigResult;
   legend: LegendConfigResult;
+  endValue?: EndValue;
+  emphasizeFitting?: boolean;
   valueLabels: ValueLabelMode;
-  layers: XYLayerConfigResult[];
+  dataLayers: DataLayerConfigResult[];
+  referenceLineLayers: ReferenceLineLayerConfigResult[];
+  annotationLayers: AnnotationLayerConfigResult[];
+  fittingFunction?: FittingFunction;
+  axisTitlesVisibilitySettings?: AxisTitlesVisibilityConfigResult;
+  tickLabelsVisibilitySettings?: TickLabelsConfigResult;
+  gridlinesVisibilitySettings?: GridlinesConfigResult;
+  labelsOrientation?: LabelsOrientationConfigResult;
+  curveType?: XYCurveType;
+  fillOpacity?: number;
+  hideEndzones?: boolean;
+  valuesInLegend?: boolean;
+  ariaLabel?: string;
+}
+
+export interface LayeredXYArgs {
```

Comment:
nit: maybe it makes sense to write this one as 
```
interface LayeredXYArgs extends Omit<XYArgs, 'dataLayers' | 'referenceLineLayers' | 'annotationsLayers'> {
    layers?: XYExtendedLayerConfigResult[];
}
```
same for `XYProps`

Reply:
I was thinking about it, but I'm sure, it is not a good practice from the perspective of future changes. Let us imagine, that those two expressions are becoming too different (and it will happen in the following PRs). They have nothing in common with each other from the perspective of the expression functions. Only the close form of the arguments. So, we don't need to create face connections between things, which just look the same, but are not. For example, in the following PRs I'm going to add the other arguments, which can't be used at `layeredXyVis`, but should be added to the `xyVis`. So, the difference will be highlighted more and more during the time. WDYT?

Reply:
Sounds good, thanks for explanation 🆗